src/functions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         4.5.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+
+/**
+ * Create global aliases for all functions in the split modules.
+ */
+require __DIR__ . '/Collection/functions_global.php';
+require __DIR__ . '/Core/functions_global.php';
+require __DIR__ . '/I18n/functions_global.php';
+require __DIR__ . '/Routing/functions_global.php';
```

Comment:
`Error/functions_global.php` is missing.

Reply:
Thanks!